src/__tests__/test-utils-wrapper.test.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import fs from 'fs';
+import path from 'path';
+import pluralize from 'pluralize';
+
+import * as components from '../../lib/components';
+import * as dom from '../../lib/components/test-utils/dom';
+import * as selectors from '../../lib/components/test-utils/selectors';
+
+jest.mock('@cloudscape-design/test-utils-core/dom');
+jest.mock('@cloudscape-design/test-utils-core/selectors');
+
+function getComponentsList() {
```

Comment:
This function already exists: https://github.com/cloudscape-design/components/blob/20fb84171d38c5ca44959cb5c70d516c1eddf3d2/src/__tests__/utils.tsx#L16C1-L30C1

Reply:
Also, you may find this one useful: https://github.com/cloudscape-design/components/blob/20fb84171d38c5ca44959cb5c70d516c1eddf3d2/src/__tests__/required-props-for-components.ts#L155